doc/Language/objects.pod6

```diff
@@ -19,30 +19,28 @@ object. That includes literals, types (type objects), code and containers.
 
 =head1 Using Objects
 
-You can use objects by calling methods on them. To call a method on an
-expression, add a dot, followed by the method name:
+To call a method on an object, add a dot, followed by the method name:
 
 =for code :allow<B L>
 L<say> "abc"B<.L<uc>>;
 # ABC
 
 This calls the L<C<uc>|uc> method on C<"abc">, which is an object of type
-L<C<Str>|Str>. If you need to supply arguments to the method, add
-parentheses after the method name with the arguments inside the parentheses:
+L<C<Str>|Str>. To supply arguments to a method, add parentheses after the
+method, with arguments inside parentheses:
```

Comment:
Would be better without the comma, changing it to ‘the arguments’. This makes it match the other words that have ‘a’, and ‘the’ in the sentence.
 “add the parentheses after the method with the arguments inside parenthesis”.